app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -886,6 +877,7 @@ class SmokeTest {
         }
     }
 
+    @Ignore("Enable after #19738 and ##19090 lands")

```

Comment:
```suggestion
    @Ignore("Enable after https://github.com/mozilla-mobile/fenix/issues/19738 and https://github.com/mozilla-mobile/fenix/issues/19090 land.")
```

Reply:
I followed the format that already existed in the other tests. :)